packages/react-data-grid/src/HeaderCellType.ts

```diff
@@ -0,0 +1,8 @@
+enum HeaderCellType {
+  SORTABLE = 0,
```

Comment:
I don't think the value matters, maybe we should just set the `SORTABLE = 'SORTABLE'`, it will be easier for debugging. as `0, 1, 2, 3` doesn't mean anything when you see the value.

https://www.typescriptlang.org/docs/handbook/enums.html - string enums

Reply:
I was thinking that maybe we could just use enum keys, with no assigned values.

>it will be easier for debugging

Shouln't be an issue with sourcemaps I think.